SpatialGDK/Source/SpatialGDK/Public/Schema/UnrealObjectRef.h

```diff
@@ -26,24 +28,6 @@ struct FUnrealObjectRef
 		, bNoLoadOnClient(bNoLoadOnClient)
 	{}
 
-	FUnrealObjectRef(const FUnrealObjectRef& In)
```

Comment:
Is copy-constructor not needed anymore?

Reply:
Using the default one, just moved it next to the empty constructor.

Reply:
Ah, I ment assignment operator

Reply:
Oh, that one just does exactly what an implicitly generated one would do, so I figured we don't need to explicitly define one.

Reply:
Maybe having a default one just as for constructors would make code self-explanatory?